scripts/vet.sh

```diff
@@ -121,7 +121,7 @@ XXXXX PleaseIgnoreUnused'
 
   # Ignore a false positive when operands have side affectes.
   # TODO(https://github.com/dominikh/go-tools/issues/54): Remove this once the issue is fixed in staticcheck.
-  noret_grep "(SA4000)" "${SC_OUT}" | not grep -ev "crl.go:\d*:\d*: identical expressions on the left and right side of the '||' operator (SA4000)"
+  noret_grep "(SA4000)" "${SC_OUT}" | not grep -v -e "crl.go:[0-9]\+:[0-9]\+: identical expressions on the left and right side of the '||' operator (SA4000)"
```

Comment:
I was getting errors with \d because it didn't match the lines, I think this is because GNU grep doesn't support \d by default.

Reply:
FYI I pinged this bug in the TODO.  It'd be nice to be able to just delete this stuff instead.

Maybe we can simplify this to just `not grep -v "crl.go:"?  I don't think the rest of it helps.